asteroid/data/librimix_dataset.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
```

Comment:
Make it importable in asteroid/data/__init__.py

Reply:
Didn't think about that thanks